### PR TITLE
[KOGITO-5009] Updated jobs-service jar name

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-5009

- Updated jobs-service infinispan jar name to new naming (due to postgresql update)
- Updated `update-maven-artifacts.py` script to use the artifacts name instead of url
- Updated jars refs

This is temporary until [KOGITO-4534](https://issues.redhat.com/browse/KOGITO-4534) is implemented

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: modules/kogito-trusty-ui/module.yaml [hunk 1]
```diff
@@ -3,9 +3,9 @@ name: org.kie.kogito.trusty.ui
 version: "2.0.0-snapshot"
 
 artifacts:
-  - name: kogito-trusty-ui-runner.jar
-    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/trusty-ui/2.0.0-SNAPSHOT/trusty-ui-2.0.0-20210419.053927-127-runner.jar
-    md5: 8207f6571c2185a8c1c3241b3f5495fe
+  - name: trusty-ui.jar
```
Comment (spolti) [inline modules/kogito-trusty-ui/module.yaml]: is there any reason to renaming it other services than job-services?
if it is really needed, the launch script of any module that you've changed the artifact name needs to be updated as well.

**Review by spolti — CHANGES_REQUESTED** [review]

File: modules/kogito-trusty-ui/module.yaml (hunk 1 above)
Comment (radtriste) [reply modules/kogito-trusty-ui/module.yaml]: oh .. right ...

I did this for the `update-maven-artifacts` script to handle property the update of artifacts...

Comment (spolti) [reply modules/kogito-trusty-ui/module.yaml]: ok :)

Comment (radtriste) [reply modules/kogito-trusty-ui/module.yaml]: I harmonized a bit the artifacts' names then :D

File: scripts/update-maven-artifacts.py [hunk 2]
```diff
@@ -114,7 +114,7 @@ def update_artifacts(service,modulePath):
 
     with open(modulePath) as module:
         data=common.yaml_loader().load(module)
-        artifact = next(filter(lambda x: service['name'] in x['url'], data['artifacts']))
+        artifact = next(filter(lambda x: service['name'] in x['name'], data['artifacts']))
```
Comment (MarianMacik) [inline scripts/update-maven-artifacts.py:118]: You may also want to update the docstring.

**Review by sutaakar — APPROVED** [review]

**Review by MarianMacik — APPROVED** [review]

File: scripts/update-maven-artifacts.py (hunk 2 above)
Comment (radtriste) [reply scripts/update-maven-artifacts.py:118]: done

**Comment by radtriste** [comment]
@MarianMacik @spolti not so easy in fact ... I would need to integrate both `in-memory` and `infinispan` jars here and decide which to run with `ENABLE_PERSISTENCE` ...

**Comment by radtriste** [comment]
> @MarianMacik @spolti not so easy in fact ... I would need to integrate both `in-memory` and `infinispan` jars here and decide which to run with `ENABLE_PERSISTENCE` ...

For now I will just reuse what has been done in `explainibility` service to select correct jar

**Comment by radtriste** [comment]
@spolti @vaibhavjainwiz can you please have a look ?

**Review by vaibhavjainwiz — APPROVED** [review]
LGTM

**Review by spolti — APPROVED** [review]

**Pull request merged**